src/modules/common/components/__tests__/Failure.spec.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Failure } from '../Failure'
+
+describe('Failure', () => {
+  const mockTranslate = key => key
+
+  it('should render a simple failure and match snapshot', () => {
+    const errorMessage = 'error message'
+    const component = shallow(<Failure errorMessage={errorMessage} t={mockTranslate} />)
+
+    expect(component.instance().props).toEqual({
+      errorMessage: errorMessage,
+      t: mockTranslate,
+      ...Failure.defaultProps
+    })
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should render a failure with goToPath and goToMessage and match snapshot', () => {
+    const error = {
+      errorMessage: 'error message',
+      goToPath: 'goTo.extras',
+      goToMessage: 'goTo.extras'
+    }
+    const component = shallow(<Failure {...error} t={mockTranslate} />)
+
+    expect(component.instance().props).toEqual({ ...error, t: mockTranslate })
+    expect(component).toMatchSnapshot()
```

Comment:
I think instead of doing a snapshot test here it would be good to just assert whether the expected text exists using: https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/text.html

Or alternatively whether a link exists to the target you expect.